Scripts/Runtime/Data/VirtualData/VirtualData.cs

```diff
@@ -1,11 +1,15 @@
 using Anvil.Unity.DOTS.Entities;
 using Anvil.Unity.DOTS.Jobs;
 using System;
+using System.Diagnostics;
 using Unity.Burst;
 using Unity.Collections;
 using Unity.Collections.LowLevel.Unsafe;
+using Unity.Entities;
 using Unity.Jobs;
 
+//TODO: DISCUSS - Namespace
```

Comment:
VirtualData is now pretty tied into Entities. It's only used by TaskDrivers which are only used by TaskDriverSystems and we only ever use `Entity` as part of the key for VirtualData. 

Thoughts on namespace re-org?

Reply:
I don't believe so anymore because the benefit in using VirtualData is via the TaskDrivers which are run by Systems. 

IF a usecase comes up in the future, we could generic VirtualData again so that the Key is not an Entity but I don't think it ever will.